### PR TITLE
Fix temperature out of range heat pump

### PR DESCRIPTION
**Pull request by rokam**
https://github.com/wuwentao/midea_ac_lan/pull/22/

### CODE REVIEW

File: midealocal/device.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ def refresh_status(self, wait_response: bool = False) -> None:
 
     def pre_process_message(self, msg: bytes) -> bool:
         if msg[9] == MessageType.query_appliance:
-            message = MessageApplianceResponse(msg)
+            message = MessageApplianceResponse(bytearray(msg))
```
Comment (chemelli74) [inline midealocal/device.py:234]: I'm not sure if this is needed.
Especially because hen we will type the library this will create an error.

Comment (rokam) [reply midealocal/device.py:234]: MessageApplianceResponse expects bytearray and msg is bytes. That was due to type error

Comment (chemelli74) [reply midealocal/device.py:234]: Please git merge with latest code, `pre_process_message` is now `bytearray` ;-)

**Review by chemelli74 — APPROVED** [review]
Nice, thx !

**Review by wuwentao — APPROVED** [review]

**Pull request merged**